boards/riscv/it8xxx2_evb/it8xxx2_evb_defconfig

```diff
@@ -14,6 +14,7 @@ CONFIG_UART_INTERRUPT_DRIVEN=y
 CONFIG_UART_CONSOLE=y
 
 CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC=32768
+CONFIG_SYS_CLOCK_TICKS_PER_SEC=8192
```

Comment:
Should you just change the default setting defined [here](https://github.com/zephyrproject-rtos/zephyr/blob/6be45c2a1809fb7e2cf8919127d86c7c68df3c29/soc/riscv/riscv-ite/it8xxx2/Kconfig.defconfig.series#L28) instead?

Reply:
Done